dissect/target/plugins/os/unix/esxi/_os.py

```diff
@@ -70,7 +72,8 @@ def __init__(self, target: Target):
 
     def _cfg(self, path: str) -> Optional[str]:
         if not self._config:
-            raise ValueError("No ESXi config!")
+            log.warning("No ESXi config!")
```

Comment:
```suggestion
            self.target.log.warning("No ESXi config!")
```
We can use the target-bound logging to get additional context on the target where the warning occurred.